config/install/field.field.config_pages.anu_lms_settings.field_beta_label.yml

```diff
@@ -0,0 +1,42 @@
+uuid: c3b2ba51-ce92-40d6-8831-8f3615fcb8e3
```

Comment:
Should be removed.

Reply:
Sorry, probably I wasn't clear saying it needs for all config files in the commit.